api/migrations/20200109220142-projectData.js

```diff
@@ -160,7 +160,7 @@ let createAgreementRecord = async function (item, project, nrptiCollection) {
     recordName: item.name,
     recordType: item.complianceDocumentType,
     // Prefer to store dates in the DB as ISO, not some random format.
-    date: moment(item.date, 'DD-MM-YYYY').toDate(),
+    dateIssued: moment(item.date, 'DD-MM-YYYY').toDate(),
```

Comment:
This breaks public LNG site.  You'll need to change the date handling for agreement documents over there as well.

Reply:
Fixed in second commit. Ran the migration and checked that it matches what shows up in dev.